mt-kahypar/partition/mapping/target_graph.h

```diff
@@ -33,12 +33,23 @@
 #include <tbb/enumerable_thread_specific.h>
 
 #ifdef __linux__
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-parameter"
```

Comment:
Can we fix the unused parameter warning?

Reply:
Well, if anyone wants to open a PR at growt, go ahead :smile:

Reply:
Ah :) No thanks. I'm actually in favor of removing growt and just using tbb_unordered_map.

Reply:
Yeah removing might make sense, but perhaps in another PR

Reply:
The `tbb::unordered_map` was 2 - 3 order of magnitudes slower for caching the Steiner trees calculations -> do **not** remove :D